.github/workflows/.lint_python.yml

```diff
@@ -0,0 +1,26 @@
+name: lint_python
+on:
+  pull_request:
+  push:
+  #  branches: [master]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8]  # , pypy3]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+      - run: pip install codespell flake8 isort mypy pytest
+      - if: matrix.python-version >= 3.6
+        run: |
+          pip install black
+          black --check . || true
+      - run: codespell --quiet-level=2 || true  # --ignore-words-list="" --skip=""
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: isort --recursive . || true
+      - run: pip install -r requirements.txt
+      - run: mypy --ignore-missing-imports . || true
+      - run: pytest . || true
```

Comment:
This also means the name is a lie, since we are no longer linting, but running the test suite.

Further more, could you rely on the tox environment we have for this very purpose?

Reply:
Why tox?  It is so old school.  Why not an automatic https://pre-commit.com ?

Reply:
Our tox environment uses pre-commit, and duplicates most of that. Also, way to win minds when you call something old school that we switched to less than 2 months ago ... :-P